contracts/PSM/ConstantPriceFeed.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity 0.8.19;
+
+import { IPriceFeed } from "../Interfaces/IPriceFeed.sol";
+import { IPriceOracle } from "../Oracles/Interfaces/IPriceOracle.sol";
+
+/// @dev Price oracle to be used for peg stability module to mint R.
+/// Returns constant price of 2 USD per token with 0 deviation.
+contract ConstantPriceFeed is IPriceFeed {
+    IPriceOracle public override primaryOracle;
+    IPriceOracle public override secondaryOracle;
+
+    uint256 public constant override lastGoodPrice = 1e18;
+    uint256 public override priceDifferenceBetweenOracles;
+
+    function setPrimaryOracle(IPriceOracle newPrimaryOracle) external override { }
+
+    function setSecondaryOracle(IPriceOracle newSecondaryOracle) external override { }
+
+    function setPriceDifferenceBetweenOracles(uint256 newPriceDifferenceBetweenOracles) external override { }
```

Comment:
nitpick - `revert NotSupported()` instead of not doing anything

Reply:
Agreed